subsys/net_core_monitor/app_core.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/types.h>
+#include <errno.h>
+#include <zephyr/kernel.h>
+#include <nrfx_ipc.h>
+#include "common.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(net_core_monitor, CONFIG_NET_CORE_MONITOR_LOG_LEVEL);
+
+#define IPC_MEM_CNT_IDX  0
+#define IPC_MEM_REAS_IDX 1
```

Comment:
Since you use the `common.h` file, why aren't those defined there and used also on the network core?

Reply:
These definitions are used only by the application core.
Access to IPC register on the network core side is by memory pointer.

Reply:
Yes, and these indexes could be used also with that `uint32_t` pointer.